pkgs/development/python-modules/pywm/default.nix

```
@@ -0,0 +1,96 @@
+{ lib,
+pkgs,
+buildPythonPackage,
+fetchFromGitHub,
+meson_0_60,
+ninja,
+pkg-config,
+wayland-scanner,
+glslang,
+libGL,
+wayland,
+wayland-protocols,
+libinput,
+libxkbcommon,
+pixman,
+seatd,
+vulkan-loader,
+xorg,
+libpng,
+libcap,
+ffmpeg,
+xwayland,
+
+imageio,
+numpy,
+pycairo,
+evdev,
+matplotlib
+}:
+
+buildPythonPackage rec {
+  pname = "pywm";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "jbuchermn";
+    repo = "pywm";
+    fetchSubmodules = true;
+    rev = "89f8de6";
```

If the source repo uses tags, bring the specific tag instead of a opaque hash string.

```suggestion
    rev = "v${version}";
```

If the source repo does not use tags, then why did you set the `version` above?